prime-angular-frontend/src/app/lib/modules/ngx-contextual-help/contextual-help/contextual-help.component.html

```diff
@@ -1,18 +1,24 @@
 <a mat-icon-button
-   color="primary"
+   [color]="color"
```

Comment:
Feels like `color` should be renamed since using `color` indicates that the entire menu might be that colour, but it only affects the menu icon.